tests/functional/kvs_many_pfx.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021 Micron Technology, Inc. All rights reserved.
+
+from contextlib import ExitStack
+
+from hse2 import hse
+
+from utility import lifecycle, cli
+
+# creates 1million + prefixes , adds data to each prefix and validates.
+
+# Test params
+# total number of prefixes
+NUM_PFXS = 1024 * 1024 + 1024
+# number of keys per prefix
+NUM_KEYS = 10
```

Comment:
can you make sure this file is formatted with black?

Reply:
yes, already did.

Reply:
Seems fishy, but if you said you did, then all is good.